### DISCUSSION TITLE
PugStats

### DISCUSSION THREAD

**Discussion by gameinside** [SmileYzn/PugMod-AMXX#35] [General]
It is necessary to change a little, as for me, there is some problem, for example, in the web so that everything counts into one statistic, and not as it is now 

https://sun9-39.userapi.com/impg/wGyQxkX2R_VqTrT-_frxcR2HjieK6RO4i-O04g/q8h1FuT4jGc.jpg?size=1679x725&quality=96&sign=15c47828038777ca04ab85bf9a84943d&type=album

**Comment by SmileYzn** [comment]
There is bots that duplicated the auth Id (steam index)

Also I did not suport non steam, duo possible duplicated steam id in servers.....

**Reply by gameinside** [reply]
> There is bots that duplicated the auth Id (steam index)
> 
> Also I did not suport non steam, duo possible duplicated steam id in servers.....

understood =(I would redo the window stats MODE, it just outputs a link to the site


and about HLTV, this is not a bad idea, since I use a plugin to control HLTV


https://forums.alliedmods.net/showthread.php?p=993138

**Reply by SmileYzn** [reply]
Stats not work duo https in page, try to disable it.

For motd that not support https, only http

**Reply by gameinside** [reply]
tnx! bro
